doc/measures_functions.md

```diff
@@ -321,6 +321,60 @@ SELECT OBS_GetMeta(
 ) FROM tablename
 ```
 
+## OBS_MetadataValidation(extent geometry, geometry_type text, metadata json, target_geoms)
+
+The ```OBS_MetadataValidation``` function peforms a validation check over
+the known issues using the extent, type of geometry and metadata we're going
+to use in the ```OBS_GetMeta``` function.
+
+#### Arguments
+
+Name | Description
+---- | -----------
+extent | A geometry of the extent of the input geometries
+geometry_type | The geometry type of the source data.
+metadata | A JSON array composed of metadata input objects.  Each indicates one desired measure for an output column, and optionally additional parameters about that column
+target_geoms | Target number of geometries.  Boundaries with close to this many objects within `extent` will be ranked highest.
+
```

Comment:
That is going to be difficult because we are now 3 engineers working on DO (@acarlon, @javitonino and myself) and waiting for one person to validate all the work is not an option, we would have a bottleneck for two reasons time zone and the amount of work).

Our workflow is to make a PR and ask one of our peers review it (In this case was you because it is a review of technical docs). We don't assign reviews to Michelle because we don't know what is her place in this puzzle but as far as I know she is part of the research team which are not in charge of DO anymore (it was assigned to engineering product team).

Summon @javisantana, @kevin-reilly or @saleiva to know more about it.

// @juanignaciosl for awareness